Projects/Server/MultiGameRunner.h

```diff
@@ -0,0 +1,33 @@
+#ifndef MULTIGAMERUNNER_H
+#define MULTIGAMERUNNER_H
+
+#include "State.h"
+#include "GameHandler.h"
+
+#include <map>
+
+class MultiGameRunner 
+{
+public:
+	MultiGameRunner(const AbstractPlayer::Ptr& player0, const AbstractPlayer::Ptr& player1, const size_t& gameSample);
+	
+	friend std::ostream& operator<<(std::ostream& os, const MultiGameRunner& multiGameRunner);
+
+	void play();
+
+	AbstractPlayer::Ptr getPlayer0() const;
+	AbstractPlayer::Ptr getPlayer1() const;
+	std::string getPlayer0Name() const;
+	std::string getPlayer1Name() const;
```

Comment:
No point exposing these 4 getters, there is no valid use case, where the client should call there.

Reply:
They are removed now.